tasks/env.py

```diff
@@ -0,0 +1,37 @@
+from os.path import dirname, abspath, join
+from multiprocessing import cpu_count
+
+PROJ_ROOT = dirname(dirname(abspath(__file__)))
+
+VERSION_FILE = join(PROJ_ROOT, "VERSION")
+LLVM_VERSION_FILE = join(PROJ_ROOT, "LLVM_VERSION")
+
+TOOLCHAIN_IMAGE_NAME = "faasm/toolchain"
+TOOLCHAIN_DOCKERFILE = join(PROJ_ROOT, "docker", "toolchain.dockerfile")
+SYSROOT_IMAGE_NAME = "faasm/sysroot"
+SYSROOT_DOCKERFILE = join(PROJ_ROOT, "docker", "sysroot.dockerfile")
+
+USABLE_CPUS = int(cpu_count()) - 1
+
+TOOLCHAIN_BIN = "/usr/local/faasm/toolchain/bin"
+SYSROOT = "/usr/local/faasm/llvm-sysroot"
+
+WASM_CC = join(TOOLCHAIN_BIN, "clang")
+WASM_AR = join(TOOLCHAIN_BIN, "llvm-ar")
+WASM_NM = join(TOOLCHAIN_BIN, "llvm-nm")
```

Comment:
There is some duplicated info in here that is held elsewhere, but consolidating everything will be covered in #6